doc/stake_pool/delegating_stake.md

```diff
@@ -32,18 +32,18 @@ The output can now be added in the `transaction` and submitted to a node.
 
 ## submitting to a node
 
-To `jcli transaction add-certificate` command can be used to add a certificate to a transaction in _finalized_ state.
+The to `jcli transaction add-certificate` command can be used to add a certificate to a transaction in _finalized_ state.
```

Comment:
_finalized_ should be _balancing_

Reply:
The `jcli transaction add-certificate` command should be used to add a certificate **before finalizing** the transaction.

Reply:
@dorin100 is right here. It would sound much better and would be more accurate.